packages/core/content-manager/admin/src/pages/EditView/components/FormLayout.tsx

```diff
@@ -3,20 +3,36 @@ import { Box, Flex, Grid } from '@strapi/design-system';
 import { EditLayout } from '../../../hooks/useDocumentLayout';
 
 import { InputRenderer } from './InputRenderer';
+import { useIntl } from 'react-intl';
 
 interface FormLayoutProps extends Pick<EditLayout, 'layout'> {}
 
 const FormLayout = ({ layout }: FormLayoutProps) => {
+  const { formatMessage } = useIntl();
+
+  const getTranslatedLabel = (label: string) => {
+    return formatMessage({
+      id: `content-manager.containers.EditPage.form.${label}`,
```

Comment:
❓  I'm wondering if we should scope the translations to a content-type and not worry about the page we are on? `content-manager.container.{content-type-name}.{attribute-name}`.

Reply:
That sounds nice, but I the same time I think it means breaking backwards compatibility? If so I think we can keep the current ids

Reply:
if we need a BC anyway then let's make the best api then 🚀 I agree the label should be the same between pages